lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -73,7 +73,7 @@
   border: $nx-border;
   border-radius: $nx-border-radius;
   color: #cd0028;
-  font-family: monospace;
+  font-family: Courier Prime, monospace;
```

Comment:
Should we make a mixin that encapsulates this line?

Reply:
> Should we make a mixin that encapsulates this line?

I thought about it but didn't think that it was worth it given how little it's used. Do you think it's worth it? For consistency if nothing else?

Reply:
> I thought about it but didn't think that it was worth it given how little it's used. Do you think it's worth it? For consistency if nothing else?

I feel like it is worth it (and easy to do) but it's not a big deal either way.

Reply:
> > I thought about it but didn't think that it was worth it given how little it's used. Do you think it's worth it? For consistency if nothing else?
> 
> I feel like it is worth it (and easy to do) but it's not a big deal either way.

https://github.com/sonatype/sonatype-react-shared-components/pull/273/commits/97344f92b4bd3a316830578ebab02ba1869e4f4d